_posts/current/blog/2020-05-07-linaro-engineering-highlights-april-2020-new-version-2.md

```diff
@@ -0,0 +1,153 @@
+
+{% include image.html path="/assets/images/content/lite.jpg" class="small-inline left" alt="lite.jpg" %}
+
+With the latest updates in TF-M and PSA, Kevin is working on a recertification with a Linaro Member board. We will post the news when that recertification is completed. 
+
+### **Linaro Consumer Group (LCG) News**
+
+Tom Gall, Director LCG
+
+This month the Linaro Consumer Group team released two blog posts highlighting work going on within the segment group and in coordination with our Member companies.
+
+{% include image.html path="/assets/images/content/LCG.jpg" class="small-inline left" alt="LCG.jpg" %}
+
+oid Open Source Project (AOSP) development boards.  John Stultz talks about the current state of support and how these devices are useful, valued members of the Android development ecosystem.  
+
+* [Update on HiKey/Hikey960 efforts in AOSP](https://www.linaro.org/blog/update-on-hikey-hikey960-efforts-in-aosp/)
+
+The second blog post is about how the effort enabling the SDM845 on the Dragonboard 845 bloomed into making possible mainline linux kernel development on a consumer form factor Android devices like the Pixel 3 and Poco F1. The efforts also highlight the effectiveness of the Android-5.4 GKI kernel and its ability to boot multiple devices from the same binary. [](https://www.linaro.org/blog/aosp-on-pixel3-pocof1-running-aosp-with-mainline-kernel-on-form-factor-devices/)
+
+**[AOSP on Pixel3/PocoF1 (Running AOSP with mainline kernels on form factor devices)](https://www.linaro.org/blog/aosp-on-pixel3-pocof1-running-aosp-with-mainline-kernel-on-form-factor-devices/)**
+
+**OTA article - Industrial Internet Consortium Journal of Innovation** Francois Ozog, Director LEDGE
+
+{% include image.html path="/assets/images/content/ledge.jpg" class="small-inline" alt="ledge.jpg" %}
+
+LEDGE has a leadership role in Over-The-Air (OTA) Special Interest Group in the Industrial Internet Consortium and was proposed to author an article on OTA and Intelligent Transport Systems (ITS). It was [published](https://www.iiconsortium.org/news/joi-articles/2020-March-JoI-Why-Are-OTA-Updates-Needed-for-ITS.pdf) on March 27th in the Journal of Innovation web page after circulation for validation amongst LEDGE members.
+
+The article exposes the challenges of OTA for current and future systems. OTA can have a significant impact in ITS. For instance, there can be an increase in peak power by 5% (resulting in 0-60mph in 2.9s instead of 3.2s). But it comes with many complexities not experienced in the mobile phone market. So many aspects of OTA in ITS need co-innovation and some form of standardization highlighted in the article.
+
+#### **[Protein Folding on Arm Devices | Helping with COVID-19](https://www.96boards.org/blog/crunch-on-arm/)** 
+
+Research by Sahaj Sarup
+
+{% include image.html path="/assets/images/content/.96boards-vertical-logo.png" class="small-inline left" alt="96boards-vertical-logo.png" %} 
```

Comment:
The image file name starts with a . but this is incorrect. Should be `path="/assets/images/content/96boards-vertical-logo.png" `